ansible/frontend.yml

```
@@ -0,0 +1,157 @@
+        var: nginx_logs.stdout_lines
+
+    - name: Restart nginx
+      systemd:
+        name: nginx
+        state: restarted
+        enabled: yes
+
+    - name: Obtain SSL certificate with certbot
+      shell: certbot --nginx -d {{ dns_name }} --non-interactive --agree-tos -m yura.tokarl@gmail.com
+      args:
+        creates: /etc/letsencrypt/live/{{ dns_name }}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)